### DISCUSSION TITLE
How to Interact with nvim-cmp's Floating Window/Popup?

### DISCUSSION THREAD

**Discussion by Demianeen** [hrsh7th/nvim-cmp#1778] [Q&A]
Hello everyone,

I've been enjoying the capabilities of nvim-cmp with Lazyvim for autocompletion in NeoVim, which has significantly enhanced my coding workflow. However, I've come across a situation where I would like to interact more directly with the floating window that appears when I'm using autocompletion — specifically, the window that shows function signatures and documentation.

Question: Is there a way to focus on or interact with the floating window created by nvim-cmp? For example, I'd like to yank text from it. I understand that these popups are generally transient and not designed for direct interaction, but any tips for working with them more effectively would be greatly appreciated.

Any advice on keybindings or plugin configurations that could facilitate these actions would be extremely helpful. I've searched around but haven't found clear guidance on how to achieve this.

Thank you for your time and assistance!

**Comment by DrKJeff16** [comment]
**Regular User Here**
I only know of the very forceful way to do so by going to normal mode and using the window switch map `<C-w>w` _(provided the floating window didn't close when escaping to Normal)_. Then you return with the same mapping.

I'll look further into that aswell.

**Reply by Demianeen** [reply]
It was a nice idea, a floating window is closing though(
